app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -198,11 +199,13 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
     final List<Task> tasks = <Task>[
       // These built-in tasks are not listed in the manifest.
       newTask('cirrus', 'cirrus', <String>['can-update-github'], false, 0),
-      newTask('mac_bot', 'chromebot', <String>['can-update-chromebots'], false, 0),
-      newTask('linux_bot', 'chromebot', <String>['can-update-chromebots'], false, 0),
-      newTask('windows_bot', 'chromebot', <String>['can-update-chromebots'], false, 0),
     ];
 
+    final List<LuciBuilder> prodBuilders = await LuciBuilder.getProdBuilders('flutter', config);
+    for (LuciBuilder builder in prodBuilders) {
+      tasks.add(newTask(builder.name, 'chromebot', <String>['can-update-github'], false, 0));
```

Comment:
shall we add and use builder.flaky instead? since flaky is being added here https://github.com/flutter/flutter/pull/63492/files

Reply:
Done.